<launch>
  <node pkg="roboteq_driver" type="roboteq_manager" name="aero_drive_roboteq_manager">
    <param name="max_rpm" value="500" />
    <param name="ppr" value="5000" />
    <param name="feedback_rate" value="5" />
    <param name="control_rate" value="10" />
    <param name="port" value="/dev/ttyACM0" />
    <param name="control_topic" value="drive_cmd" />
    <param name="info_topic" value="aero_drive_feedback" />
    <param name="pause_topic" value="/pause" />
  </node>
  <node pkg="aero_base" type="aero_drive_node" name="aero_drive_node">
    <param name="rotations_per_meter" value="45" />
    <param name="base_width" value="0.6" />
    <param name="roboteq_manager_topic" value="drive_cmd" />
    <param name="twist_topic" value="cmd_vel" />
  </node>
</launch>